config/settings.yml

```diff
@@ -4625,6 +4733,22 @@ weapons:
     name: LAU-131 - 7x WTU1B (Practice)
     type: rockets
     weight: 225
+  lau10:
+    name: 4x ZUNI MK 71
+    type: rockets
+    weight: 1144
+  lau61:
+    name: 19x 2.75' ROCKETS MK151 HE
+    type: rockets
+    weight: 816
+  lau68151:
+    name: 7x 2.75' ROCKETS M151 HE
+    type: rockets
+    weight: 428
+  lau685:
+    name: 7x 2.75' ROCKETS MK5 HE
+    type: rockets
+    weight: 401
```

Comment:
ok, das hier ist komisch. in der A-10 gibt es auch schon den LAU-68 mit jeweils 7x M151 und MK5. Da wiegt das nur ca. 227 lbs:

```
  lau68mk5:
    name: LAU-68/A - 7x MK5 (HE)
    type: rockets
    weight: 227

und

  lau131m151:
    name: LAU-131 - 7x M151 (HE)
    type: rockets
    weight: 225
```
Wieso wiegt das hier anders? ist da evtl noch ein Pylon mit dabei, der das Extragewicht verursacht?